llama_index/multi_modal_llms/generic_utils.py

```diff
@@ -0,0 +1,15 @@
+from llama_index.readers import SimpleDirectoryReader
+from llama_index.schema import ImageDocument
+
+
+def load_image_urls(image_urls: list[str]) -> list[ImageDocument]:
```

Comment:
just checking, this is not used in this PR right? more as an utility for people to pick up when necessary?

Reply:
not it is used in the notebook example for Fuyu model

Reply:
nvm it's used in the notebook.